kedge/auth.yml

```diff
@@ -0,0 +1,18 @@
+name: auth
+containers:
+- image: docker.io/fabric8/fabric8-auth:latest
+  env:
+  - name: AUTH_WIT_URL
+    value: "http://192.168.42.1:8080"
+  - name: AUTH_DEVELOPER_MODE_ENABLED
+    value: "true"
+  - name: AUTH_POSTGRES_HOST
+    value: "db-auth"
```

Comment:
Nice.

Reply:
this was possible as `auth` and `db-auth` both are on OpenShift.